localenv/README.md

```diff
@@ -162,34 +157,36 @@ pnpm localenv:compose down --volumes
 
 ### Usage
 
-#### Postman & Open Payments APIs
+#### Bruno & Open Payments APIs
 
-The Open Payments APIs can be interacted with using the Postman collection ([Open Payments APIs](https://www.postman.com/interledger/workspace/interledger/folder/22855701-1b204bc1-c8e5-44d4-bab9-444d7204b15a?ctx=documentation) and [Open Payments Auth APIs](https://www.postman.com/interledger/workspace/interledger/folder/22855701-ae80b96d-4d25-42b9-94fa-8ed17f0e5ed9?ctx=documentation)). It is configured to use the default endpoints of the local environment.
+The Open Payments APIs can be interacted with using the [Bruno collection](https://github.com/interledger/rafiki/main/bruno/collections/Interledger) and its ([Open Payments APIs](https://github.com/interledger/rafiki/main/bruno/collections/Interledger/Open%20Payments%20APIs) and [Open Payments Auth APIs](https://github.com/interledger/rafiki/main/bruno/collections/Interledger/Open%20Payments%20Auth%20APIs)). It is configured to use the default endpoints of the local environment.
```

Comment:
Maybe "The Bruno collection can be used to interact with the Open Payments API and Open Payments Auth API"?

Reply:
I'm good with that description, but I think I asked @sabineschaller during the Rafiki work week whether these were all separate APIs or just a single Open Payments API and, I think she said single..? Sabine can you refresh my memory?

Reply:
Thank you for making me be more precise @melissahenderson 
We technically have 3 Open Payments APIs now:
- resource server API
- auth server API
- wallet address API

I don't think we make the distinction between resource server and wallet address API in the collection yet, because we have not moved the wallet address to be hosted on its own server. 

Nevertheless, I think the correct wording would be 

> The Open Payments APIs can be interacted with using the Bruno collection. We provide all the resource and auth server API endpoints.